entries/iteration.qmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "Time-constrained iterative workflow"
+date: last-modified
+---
+
+We take inspiration from some aspects of [Agile Software
+Development](https://agilemanifesto.org/) and
+[Scrum](https://www.scrum.org/resources/what-scrum-module), especially this idea
+of time-boxed periods ("iterations") of work that follow a specific pattern and process
+for incrementally building or improving things.  An iteration for us follows
+this general sequence of steps:
+
+- Team lead creates a GitHub Project Board for the upcoming iteration using the "Iteration" template.
+- The first meeting of the iteration is to discuss and plan the specifics of the iteration.
+    - Decide on the end output (the "increment" or "milestone").
+    - Brainstorm and write out as many tasks as possible that are needed to complete the increment or milestone, or select them from the current list. A task is a GitHub Issue.
+    - Each team member self-assigns themselves to a task (self-assign to Issue) and estimates the time spent on them by using the Iteration's Roadmap Board.
+    - Decide on how long the iteration should or will take and set a date for the "last iteration meeting".
+- Throughout the iteration, team members work on their assigned tasks, review Pull Requests from others, add more Issues if required (either adding to the current iteration or putting it for later iterations), and comment where required or mentioned in Issues. Update meetings on progress happen twice a week (see the post on [Update Meetings](update-meetings.md) for details on how we run them). 
```

Comment:
I'm not sure what "putting it for later iterations" mean? Is it to add issues that can be put in a later iteration instead of being a part of the current one? 
Also not sure I understand "comment where required or mentioned in Issues" -> maybe add a bit more context to that? :)

Reply:
Do you mean that we note issues for future iterations? With regards to the comments, should that be comments and reviews, rather than just comments?